### PR TITLE
feat: table sticky headers and style adjustments

### PR DESCRIPTION
**Pull request by alfonsobries**
## Summary

- Adds the styling used to create sticky headers (https://github.com/ArkEcosystem/explorer.ark.io/pull/703)
- Updates the table header component so it accepts the same responsive related props that we use with the regular cells.
- Updates to the table styling to match the style guide (see before and after screenshot the changes are on the  spacing/aligment on the sides) :

### Before:

image

### After:

image

## Checklist

-   [x] I checked my UI changes against the design and there are no notable differences
-   [x] I checked my UI changes for any responsiveness issues
-   [x] I checked my (code) changes for obvious issues, debug statements and commented code
-   [x] I provided a screenshot of my changes to the component _(if applicable)_
-   [ ] I regenerated the `icons.html` file and checked if my newly added icon is shown correctly _(if necessary)_
-   [x] I added an explanation on how to use the component to the readme _(if necessary)_
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [ ] Ready to be merged

### CODE REVIEW

File: resources/assets/css/_tables.css [hunk 1]
```diff
@@ -663,3 +564,77 @@
     > div.table-cell-bg::before {
     @apply bg-transparent;
 }
+
+/* Table with sticky headers you will need to manually override the `top` property for `.table-container table.sticky-headers th` according to your needs */
```
Comment (alexbarnsley) [inline resources/assets/css/_tables.css:568]: @alfonsobries could you split this into 2 lines, please. It's good to try and keep lines shorter (than this) so you don't get the horizontal scroll on GH

File: resources/views/tables/header.blade.php [hunk 2]
```diff
@@ -30,5 +30,9 @@
         . ($firstOn === 'xl' ? ' first-cell first-cell-xl' : '')
         . ' ' . $class
 ]) }}>
-    @lang($name)
+    @if ($name != '')
```
Comment (alexbarnsley) [inline resources/views/tables/header.blade.php:33]: Can probably just be `@if ($name)` here

**Review by alexbarnsley — COMMENTED** [review]
A couple of minor comments @alfonsobries

**Comment by alexbarnsley** [comment]
@ItsANameToo "approved" - some comments but no show-stoppers

**Comment by alfonsobries** [comment]
@alexbarnsley ✅

**Review by ItsANameToo — APPROVED** [review]

**Pull request merged**